templates/default/lib/getPageData.js

```diff
@@ -0,0 +1,15 @@
+export async function getPageData(dataProperties) {
+  const data = {};
+  for (const dataProperty of dataProperties) {
+    try {
+      const dataGetter = (await import(`../content/getters/${dataProperty}.js`))
```

Comment:
this is cool - i didn't realize one could do dynamic imports like this. This may help us refactor `siteConfig.js` to have a dynamic import location for the `config.js` - see https://github.com/flowershow/flowershow/issues/7